ampalibe/utils.py

```diff
@@ -61,10 +75,12 @@ def trt_payload_in(payload):
 
 
 def trt_payload_out(payload):
-    '''
-        text payload processing
-        payload object to text
-    '''
+    """
+    A function that will  take as parameter the object payload and as output a string .
```

Comment:
I think more explain what the function does. not just mentioning parameter and output.
Something more like :  `Function to convert payload object to a string version`
To specify parameter & return : 
```
    :param payload : <description>
    :return  payload : <description>
```